package.json

```diff
@@ -108,6 +108,8 @@
     ]
   },
   "dependencies": {
+    "@changesets/changelog-github": "^0.4.6",
+    "@changesets/cli": "^2.24.3",
```

Comment:
[suggest] Should the new packages be in `devDependencies`?

Reply:
Sorry, I forgot to use `-d` flag.